server/spec/consumer_resource_dev_spec.rb

```diff
@@ -88,7 +88,8 @@ def auto_attach_and_verify_dev_product(expected_product_id, old_ent_id=nil)
     consumer = @user.register(random_string('system'), type=:system, nil, facts = {:dev_sku => "dev_product"}, @username,
               @owner['key'], [], [], nil, [], nil, [], created_date)
     consumer_client = Candlepin.new(nil, nil, consumer['idCert']['cert'], consumer['idCert']['key'])
-    expected_error = "Unable to attach subscription for the product 'dev_product': Subscriptions for dev_product expired on:"
+    expected_error = "Unable to attach subscription for the product \"dev_product\": Subscriptions for
+    dev_product expired on:"
```

Comment:
This line break is causing the failures with the spec tests

Reply:
Fixed